--- /export/home/oyvindsc/sleep2.h	2015-04-29 14:18:16.746277259 +0200
+++ /usr/lib/avr/include/avr/sleep.h	2015-04-29 13:22:02.805858278 +0200
@@ -427,22 +427,50 @@
     } while(0)
 
 #elif defined(__AVR_ATxmega16A4__) \
+|| defined(__AVR_ATxmega16A4U__) \
+|| defined(__AVR_ATxmega16C4__) \
 || defined(__AVR_ATxmega16D4__) \
 || defined(__AVR_ATxmega32A4__) \
+|| defined(__AVR_ATxmega32A4U__) \
+|| defined(__AVR_ATxmega32C3__) \
+|| defined(__AVR_ATxmega32C4__) \
+|| defined(__AVR_ATxmega32D3__) \
 || defined(__AVR_ATxmega32D4__) \
+|| defined(__AVR_ATxmega8E5__) \
+|| defined(__AVR_ATxmega16E5__) \
+|| defined(__AVR_ATxmega32E5__) \
 || defined(__AVR_ATxmega64A1__) \
 || defined(__AVR_ATxmega64A1U__) \
 || defined(__AVR_ATxmega64A3__) \
+|| defined(__AVR_ATxmega64A3U__) \
+|| defined(__AVR_ATxmega64A4U__) \
+|| defined(__AVR_ATxmega64B1__) \
+|| defined(__AVR_ATxmega64B3__) \
+|| defined(__AVR_ATxmega64C3__) \
 || defined(__AVR_ATxmega64D3__) \
+|| defined(__AVR_ATxmega64D4__) \
 || defined(__AVR_ATxmega128A1__) \
 || defined(__AVR_ATxmega128A1U__) \
 || defined(__AVR_ATxmega128A3__) \
+|| defined(__AVR_ATxmega128A3U__) \
+|| defined(__AVR_ATxmega128A4U__) \
+|| defined(__AVR_ATxmega128B1__) \
+|| defined(__AVR_ATxmega128B3__) \
+|| defined(__AVR_ATxmega128C3__) \
 || defined(__AVR_ATxmega128D3__) \
+|| defined(__AVR_ATxmega128D4__) \
 || defined(__AVR_ATxmega192A3__) \
+|| defined(__AVR_ATxmega192A3U__) \
+|| defined(__AVR_ATxmega192C3__) \
 || defined(__AVR_ATxmega192D3__) \
 || defined(__AVR_ATxmega256A3__) \
+|| defined(__AVR_ATxmega256A3U__) \
+|| defined(__AVR_ATxmega256C3__) \
 || defined(__AVR_ATxmega256D3__) \
-|| defined(__AVR_ATxmega256A3B__)
+|| defined(__AVR_ATxmega256A3B__) \
+|| defined(__AVR_ATxmega256A3BU__) \
+|| defined(__AVR_ATxmega384C3__) \
+|| defined(__AVR_ATxmega384D3__)
 
     #define SLEEP_MODE_IDLE         (0)
     #define SLEEP_MODE_PWR_DOWN     (SLEEP_SMODE1_bm)
